packages/graphql/src/classes/NodeDirective.ts

```diff
@@ -37,42 +37,15 @@ export class NodeDirective {
         if (!typeName) {
             throw new Neo4jGraphQLError("Could not generate label string in @node directive due to empty typeName");
         }
-        const labels = this.getLabels(typeName, context).map((l) => this.escapeLabel(l));
+        const labels = this.getLabels(typeName, context).map((label) => Cypher.utils.escapeLabel(label));
```

Comment:
I'm not 100% sure about this change, are we using this somewhere else? Why is this part of the new translation layer?

Reply:
This is to unify the dynamic labels logic between the two implementations "Schema Model" and "Node" to avoid having duplicate code achieving the exact same result; `this.escapeLabel` was literally just invoking the `Cypher.utils.escapeLabels`!